tslint.json

```diff
@@ -16,7 +16,8 @@
     "array-type": [true, "generic"],
     "interface-name": [true, "always-prefix"],
     "arrow-return-shorthand": true,
-    "arrow-parens": false
+    "arrow-parens": false,
```

Comment:
What is this?

Reply:
It is a rule that allows:
```ts
e => someFuntionsBody;
```

and don't require parantheses like this:
```ts
(e) => someFunctionBody;
```